index.html

```diff
@@ -7013,7 +7016,7 @@ <h5>Script Injection</h5>
 	      when generating HTML: with care. 
 	      <span class="rfc2119-assertion" id="sec-inj-sanitize">
 		      Strings sourced from TDs MUST either be sanitized using a carefully vetted HTML
```

Comment:
There was a problem with the markup tooling that prevents having two assertions in the same <span>. Please check with @mmccool  on how to handle that.